coriolisclient/cli/shell.py

```diff
@@ -356,7 +359,7 @@ def _setup_logging():
 
 
 def main(argv=sys.argv[1:]):
-    _setup_logging()
+    _setup_logging(debug=('--debug' in argv))
```

Comment:
Please note that "arbitrary arg processing" (i.e. outside of using `argparse` or some library which in turn uses `argparse`) is risky, as various subcommand classes could error out saying "unrecognized argument --debug" since it will get passed to them as well.

We should either:
1. exclude the arg from the list of args actually passed down the stack
2. properly define the arg and ensure that all subcommands work

In my view, the first option would be the way cleaner one.